prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -110,10 +110,10 @@ do {                                                                	\
 	 PEND_SEND_IS_LIMITED(_ep))
 
 struct fi_ibv_rdm_header {
-	uint64_t imm_data;          // TODO: not implemented
+/*	uint64_t imm_data; TODO: not implemented */
```

Comment:
It's changing the wire protocol.  If you update one side but not the other, the two sides will disagree on the fields that they access and break.  Before we can claim that verbs RDM support is truly supported, the wire protocol needs to be fully defined and forward/backward compatible, with proper versioning.